src/pair_rann.h

```diff
@@ -0,0 +1,171 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+/*  ----------------------------------------------------------------------
+   Contributing authors: Christopher Barrett (MSU) barrett@me.msstate.edu
+   	   	   	   	   	     Doyl Dickel (MSU) doyl@me.msstate.edu
+    ----------------------------------------------------------------------*/
+/*
+“The research described and the resulting data presented herein, unless
+otherwise noted, was funded under PE 0602784A, Project T53 "Military
+Engineering Applied Research", Task 002 under Contract No. W56HZV-17-C-0095,
+managed by the U.S. Army Combat Capabilities Development Command (CCDC) and
+the Engineer Research and Development Center (ERDC).  The work described in
+this document was conducted at CAVS, MSU.  Permission was granted by ERDC
+to publish this information. Any opinions, findings and conclusions or
+recommendations expressed in this material are those of the author(s) and
+do not necessarily reflect the views of the United States Army.​”
+
+DISTRIBUTION A. Approved for public release; distribution unlimited. OPSEC#4918
+ */
+
+#ifdef PAIR_CLASS
+
+PairStyle(rann,PairRANN)
+
+#else
+
+#ifndef LMP_PAIR_RANN
+#define LMP_PAIR_RANN
+
+#define MAXLINE 1024
```

Comment:
the define and the following include files should be moved from the header to the .cpp file. only the essential include statements like for pair.h should remain. Please see https://github.com/lammps/lammps/blob/master/doc/include-file-conventions.md for some explanations.

Reply:
Done.